cmd/lukso/start.go

```diff
@@ -190,12 +190,17 @@ func startPrysm(ctx *cli.Context) error {
 
 func startValidator(ctx *cli.Context) error {
 	log.Info("🔄  Starting Validator")
-	validatorFlags, ok := prepareValidatorStartFlags(ctx)
-	if !ok {
-		return errFlagPathInvalid
+	validatorFlags, err := prepareValidatorStartFlags(ctx)
+	if err != nil {
+		return err
+	}
+	if !fileExists(fmt.Sprintf("%s/direct/accounts/all-accounts.keystore.json", ctx.String(validatorKeysFlag))) { // path to imported keys
```

Comment:
i feel that this string "/direct/accounts/all-accounts.keystore.json" should go in a variable no?